front/src/components/table/table-header/SortAndFilterBar.tsx

```diff
@@ -24,6 +24,26 @@ const StyledBar = styled.div`
   height: 40px;
 `;
 
+const StyledFilterButton = styled.button`
```

Comment:
I feel naming should be StyledCancelButton

Reply:
Good point. It was poorly named because I changed my mind mid-writing the code and never updated the name ;)